### DISCUSSION TITLE
Could we get a @dylanseeger design audit for desktop UI? I'd be willing to help make PRs for the UI updates

### DISCUSSION THREAD

**Discussion by nth-chile** [glushchenko/fsnotes#1572] [General]
I think I can help with this! @dylanseeger if you'd be interested in performing a design audit, like you did to the iOS app

**Comment by dylanseeger** [comment]
I find the Mac app to be in much better shape than the iOS app was. When configured with vertical layout enabled and the sidebar closed, it is essentially a Notational Velocity, which is all I've ever wanted.

I recommended a switch to SF Symbols (#1509) back in April. Pursuing that would help a lot. Otherwise, I would focus almost entirely on bug fixes. The app doesn't need any new features, it needs to be reliable.

**Reply by nth-chile** [reply]
I agree that it looks good, but I think there are some minor issues that could make a big difference overall. But I'm not a designer so I don't always have the best answers to those issues.

1. There's a brown background at the bottom
2. Encrypted folder icon is black
3. Selected tag is light blue on white, which seems like low contrast
4. "iCloud Drive" looks like it's floating without context. I really like Finder's method of titling things in the sidebar. ("Favorites", "iCloud", Locations", "Tags"). Adding screenshot of that at the bottom of this post

<img width="182" alt="Screenshot 2023-07-12 at 4 57 43 PM" src="https://github.com/glushchenko/fsnotes/assets/22358233/fc8ea1be-ef25-478c-a396-5d3aeb72177f">

1. Is the color of that lock a one-off or is it in the color scheme?
2. The color of the lightning bolt icon is low contrast with background. Maybe we could replace it with the new logo, but just the note lines without the rectangle background. The color could be transparent black, like a silhouette

<img width="775" alt="Screenshot 2023-07-12 at 5 01 50 PM" src="https://github.com/glushchenko/fsnotes/assets/22358233/c884eab4-72bc-4501-8afd-142565503643">

1. Green locks here as well could be SF Symbols

<img width="223" alt="Screenshot 2023-07-12 at 5 03 59 PM" src="https://github.com/glushchenko/fsnotes/assets/22358233/31c723a3-198b-4a25-aec3-f86431328917">

Maybe we could make use of system accent color. I think it is yellow by default, but you can change your system's accent color in system settings.

Love these section titles in Finder

<img width="146" alt="Screenshot 2023-07-12 at 5 08 51 PM" src="https://github.com/glushchenko/fsnotes/assets/22358233/8cf3db4e-0702-469f-b33a-17696666451c">

**Comment by nth-chile** [comment]
Todo items have a slightly bigger line space to make room for the large checkbox

<img width="467" alt="Screenshot 2023-07-12 at 5 21 20 PM" src="https://github.com/glushchenko/fsnotes/assets/22358233/c0d7529c-08de-4132-bd7b-763ca51bbc97">

**Comment by glushchenko** [comment]
@dylanseeger 

> Otherwise, I would focus almost entirely on bug fixes. The app doesn't need any new features, it needs to be reliable.

Good suggestions, but honestly I miss the bug reports. I try to fix existing problems promptly. And I remember about #1509 ;)

@nth-chile I can't reproduce it. Сan you put the details in a separate ticket?

**Reply by nth-chile** [reply]
Sure, I'll post them as individual tickets soon. I should be ready to start contributing this winter 😎
